.github/quarkus-bot.yml

```diff
@@ -290,8 +290,10 @@ triage:
         - extensions/cache/
         - integration-tests/cache/
     - labels: [area/cli]
+      title: "cli|qs"
       directories:
         - devtools/cli/
+      notify: [maxandersen]
```

Comment:
I think we should add `ebullient` and me given we are currently working on the subject.

Reply:
```suggestion
      notify: [maxandersen,ebullient,gsmet]
```